packages/frameworks/test/frameworks.unit.test.ts

```diff
@@ -62,16 +62,40 @@ const SchemaSettings = {
   ],
 };
 
+const RouteSchema = {
+  type: 'array',
+  items: {
+    properties: {
+      src: { type: 'string' },
+      dest: { type: 'string' },
+      status: { type: 'number' },
+      handle: { type: 'string' },
+      headers: { type: 'object' },
+      continue: { type: 'boolean' },
+    },
+  },
+};
+
 const Schema = {
   type: 'array',
   items: {
     type: 'object',
-    required: ['name', 'slug', 'logo', 'description', 'settings'],
+    additionalProperties: false,
```

Comment:
Adding a new property to a framework entry will now cause this test to fail, forcing the author to add a type to the schema for it.